File: 412/main.py
# FizzBuzz
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        result = []
        for i in range(1, n+1):
            add = ""
            if i % 3 == 0:
                add += "Fizz"
            if i % 5 == 0:
                add += "Buzz"
            if i % 5 != 0 and i % 3 != 0:
                add = str(i)
            result.append(add)
        return result

ans = Solution()
print(ans.fizzBuzz(18))

File: 412/main.go
package main

import (
	"bytes"
	"fmt"
	"strconv"
)

func main() {
	fmt.Println(fizzBuzz(15))
}

func fizzBuzz(n int) []string {
	result := []string{}
	for val := 1; val <= n; val++ {
		var str bytes.Buffer
		if val%3 == 0 {
			str.WriteString("Fizz")
		}
		if val%5 == 0 {
			str.WriteString("Buzz")
		}
		if len(str.String()) == 0 {
			str.WriteString(strconv.Itoa(val))
		}
		result = append(result, str.String())
	}
	return result
}
